pkg/kubectl/resource_printer.go

```diff
@@ -880,13 +880,16 @@ func printIngress(ingress *extensions.Ingress, w io.Writer, options PrintOptions
 		}
 	}
 
-	if _, err := fmt.Fprintf(w, "%s\t%v\t%v\t%v\n",
+	if _, err := fmt.Fprintf(w, "%s\t%v\t%v\t%v",
```

Comment:
A strange side-effect of appendLabels is that it always appends a new line. Hrm. I guess it works out in this scenario.